src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +506,35 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
+
+    /**
+     * Lists all the forks of this repository.
+     */
+    public PagedIterable<GHRepository> listForks() {
+      return listForks(null);
+    }
+
+    /**
+     * Lists up all the forks of this repository, sorted by the given sort order.
+     */
+    public PagedIterable<GHRepository> listForks(final Sort sort) {
```

Comment:
There is not a single `@CheckForNull` in the entire codebase so far. If you want to start introducing them, go ahead, but that was not the scope of this pull request.
Regarding Javadoc, I was under the impression that both methods I added actually have Javadoc comments, but maybe I miscounted.